src/YT/YT_Inline.cpp

```diff
@@ -237,19 +236,27 @@ void YT_Inline()
    ParticleList[0].coor_z   = "ParPosZ";
 
    // Set get attribute function
-   ParticleList[0].get_attr = Get_ParticleAttribute;
+   ParticleList[0].get_par_attr = Get_ParticleAttribute;
 
 #  endif // #ifdef PARTICLE
 
 // 4. prepare local patches for libyt
    YT_AddLocalGrid( NField, FieldList, pc);
 
-// 5. perform yt inline analysis
-   if ( yt_inline_argument( "yt_inline_argument", 1, "\'Dens\'" ) != YT_SUCCESS )   Aux_Error( ERROR_INFO, "yt_inline_argument() failed !!\n" );
-   if ( yt_inline( "yt_inline" ) != YT_SUCCESS )   Aux_Error( ERROR_INFO, "yt_inline() failed !!\n" );
+// 5-1. perform yt inline analysis
+   if ( yt_run_FunctionArguments( "yt_inline_inputArg", 1, "\'Dens\'" ) != YT_SUCCESS )
+       Aux_Error( ERROR_INFO, "yt_run_FunctionArguments() failed !!\n" );
+   if ( yt_run_Function( "yt_inline" ) != YT_SUCCESS )
+       Aux_Error( ERROR_INFO, "yt_run_Function() failed !!\n" );
+
+#ifdef SUPPORT_LIBYT_INTERACTIVE
+// 5-2. activate libyt interactive mode
+   if ( yt_run_InteractiveMode("LIBYT_STOP") != YT_SUCCESS )
```

Comment:
Do you prefer letting each simulation code define its own file name (i.e., `LIBYT_STOP` here) or fixing it to be `LIBYT_STOP` by `libyt`?

Reply:
Yes